eth/vm/forks/frontier/__init__.py

```diff
@@ -36,7 +37,8 @@
 
 def make_frontier_receipt(base_header: BlockHeaderAPI,
                           transaction: SignedTransactionAPI,
-                          computation: ComputationAPI) -> ReceiptAPI:
+                          computation: ComputationAPI,
+                          gas_used: int) -> ReceiptAPI:
```

Comment:
There's a little ambiguity here for a naive reader: is this the transaction's gas used, or the cumulative gas used in the header (and before or after the transaction). Maybe something like:
```suggestion
                          new_cumulative_gas_used: int) -> ReceiptAPI:
```

Or, if you can't stomach the name, at least a docstring, though I like a self-documenting name better. :)